yaqd-wright-filter-wheels/wright_filter_wheels_continuous/_yaqd_wright_filter_wheels_continuous.py

```diff
@@ -0,0 +1,76 @@
+__all__ = ["YaqdWrightFilterWheelsContinuous"]
+
+import asyncio
+from typing import Dict, Any, List
+
+from yaqd_core import ContinuousHardware, aserial
+
+from .__version__ import __branch__
+
+class YaqdWrightFilterWheelsContinuous(ContinuousHardware):
+    _kind = "yaqd-wright-filter-wheels-continuous"
+    _version = "0.1.0" + f"+{__branch__}" if __branch__ else ""
+    traits: List[str] = ["uses-uart","uses-serial","is-homeable"]
+    defaults: Dict[str, Any] = {"baud_rate": 57600}
+
+    def __init__(self, name, config, config_filepath):
+        super().__init__(name, config, config_filepath)
+        
+        self._serial_port = aserial.ASerial(config["serial_port"], config["baud_rate"])
+
+
+    def _load_state(self, state):
+        """Load an initial state from a dictionary (typically read from the state.toml file).
+
+        Must be tolerant of missing fields, including entirely empty initial states.
+
+        Parameters
+        ----------
+        state: dict
+            The saved state to load.
+        """
+        super()._load_state(state)
+        # This is an example to show the symetry between load and get
+        # If no persistent state is needed, these unctions can be deleted
+        self.value = state.get("value", 0)
+
+    def get_state(self):
+        state = super().get_state()
+        state["value"] = self.value
+        return state
+
+    def _set_position(self, motor, position):
```

Comment:
motor index should be set in the config, there is one daemon per filter wheel (not just one for the whole box)